tests/test_suites/LongSystemTests/test_build_while_auto_recovery.sh

```diff
@@ -2,11 +2,11 @@ timeout_set 1 hour
 assert_program_installed git
 assert_program_installed cmake
 
-CHUNKSERVERS=2 \
+CHUNKSERVERS=3 \
```

Comment:
Why is needed to change the number of chunkservers?

Reply:
I guess for the space needed for the generated objects during building. Antuan, please clarify.

Reply:
I'm not sure why, even when theoretically only 2 chunkservers are needed if standard 2 goal replication is set, after trying different parameters, this was the minimum amount of chunkservers that made the test passed.

Reply:
Yes, sure. Let's move on!!!